newrelic/hooks/mlmodel_openai.py

```diff
@@ -268,10 +346,21 @@ def create_chat_completion_message_event(
     response_id,
     request_id,
     conversation_id,
+    choices_message,
 ):
     message_ids = []
     for index, message in enumerate(message_list):
```

Comment:
So we'd have two for loops here instead of just the one; one for choices and one for messages and the choices would have the is_response attr set.